.gitlab-ci.yml

```diff
@@ -87,3 +87,44 @@ container:
     expire_in: 5 weeks
     paths:
       - release.env
+chaos:
+  stage: test
+  image: ubuntu
+  script:
+    - apt-get update && apt-get install docker.io -y && apt-get install curl -y && apt-get install wget
+    - service docker start
+    - docker version
+    - wget https://dl.google.com/go/go1.13.5.linux-amd64.tar.gz && tar zxf go1.13.5.linux-amd64.tar.gz -C /usr/local/
+    - export PATH=$PATH:/usr/local/go/bin
+    - GO111MODULE="on" go get sigs.k8s.io/kind@v0.6.1 && export PATH=$PATH:~/go/bin
```

Comment:
Any reason for not using 0.7.0, @rajdas98?

Reply:
No reason @taylor . I upgraded to 0.7.0